cyclus_input/reactors/Arkansas_Nuclear_1.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="no" ?>
```

Comment:
Nope, after testing with and without these, there seems to be no errors. I will remove these.